lib/sqlalchemy/sql/selectable.py

```diff
@@ -4492,7 +4497,7 @@ def get_plugin_class(cls, statement: Executable) -> Type[SelectState]:
 
     def __init__(
         self,
-        statement: Select[Any],
+        statement: Select[Unpack[Tuple[Any, ...]]],
```

Comment:
**Michael Bayer** (zzzeek) wrote:

same

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4986

Reply:
Done